spec/models/appointment_summary_spec.rb

```diff
@@ -169,4 +169,27 @@
       it { is_expected.to eq(appointment_summaries[2, 3]) }
     end
   end
+
+  context 'notification ID' do
+    subject { create(:appointment_summary, requested_digital: true, email: 'test@test.com') }
+    before { subject.update(notification_id: '123') }
+
+    it 'resets the notification ID if the email changes' do
+      expect do
+        subject.update(email: 'new@test.com')
+      end.to change { subject.notification_id }.from('123').to('')
+    end
+
+    it 'does not resets the notification ID if the email doesnt changes' do
+      expect do
+        subject.update(email: subject.email)
+      end.not_to change { subject.notification_id }
+    end
+
+    it 'does not resets the notification ID if other fields changes' do
```

Comment:
`does not reset the notification ID if fields other than the email address are changed`?